test/core/relaxed-simd/i32x4_relaxed_trunc.wast

```diff
@@ -73,7 +75,8 @@
                        (v128.const f64x2 -1.0 4294967296.0))
                ;; out of range -> saturate or UINT32_MAX
                (either (v128.const i32x4 0 0xffffffff 0 0)
-                       (v128.const i32x4 0xffffffff 0xffffffff 0 0)))
+                       (v128.const i32x4 0xffffffff 0xffffffff 0 0)
+                       (v128.const i32x4 0xfffffffe 0 0 0)))
```

Comment:
IIRC when I was stepping in the debugger: 0xfffffffe comes from `addpd` after -1.0 + 4503599627370496.0

Reply:
Is the addpd from implementing this relaxed instruction using the simd128 one? (i32x4.trunc_sat_f64x2_s_zero)

Reply:
V8 generates:
```
0x9923509f796    16  c4e37909c00b         vroundpd xmm0,xmm0,0xb
0x9923509f79c    1c  49ba90e7fa2001000000 REX.W movq r10,0x120fae790
0x9923509f7a6    26  c4c1795802           vaddpd xmm0,xmm0,[r10]
0x9923509f7ab    2b  c4c178c6c788         vshufps xmm0,xmm0,xmm15,0x88
```
(notice that liftoff generates trunc_sat_f64x2_s_zero code, [not relaxe](https://bugs.chromium.org/p/v8/issues/detail?id=13849)d)

Reply:
I guess that implementation comes from https://github.com/WebAssembly/simd/pull/383 "x86/x86-64 processors with AVX instruction set".
That codegen should return a 0. If it doesn't then maybe there's a bug in the generated code i think.

Using the trunc_sat instruction to implement the relaxed instruction is always correct (for all the relaxed instructions). This is so that in deterministic mode, we can fall back to SIMD128 instructions.

Reply:
This discussion seems to conclude that 0xfffffffe is a bug, yet it made it into #144. I'm confused.